scripts/setup.sh

```diff
@@ -340,6 +341,66 @@ else
   GCP_SCHEDULER_CRON="*/${QUERY_INTERVAL_MIN} * * * *"
 fi
 
+# If --upgrade option is not set, all gcp extensions are downloaded from the cluster to get configuration of gcp services for version that is currently active on the cluster.
+if [[ "$UPGRADE_EXTENSIONS" != "Y" ]]; then
+  echo "- downloading active extensions from Dynatrace"
+  mkdir -p ../extensions_from_cluster
+  cd ../extensions_from_cluster || exit
+  EXTENSIONS_TO_DOWNLOAD="com.dynatrace.extension.google"
+  readarray -t EXTENSIONS_FROM_CLUSTER_ARRAY <<<"$( echo "${EXTENSIONS_FROM_CLUSTER}" | sed 's/ /\n/' | grep "$EXTENSIONS_TO_DOWNLOAD" )"
+  for i in "${!EXTENSIONS_FROM_CLUSTER_ARRAY[@]}"; do
```

Comment:
use different name then `i`

Reply:
I left it as is because it's the variable for index